build_examples.sh

```diff
@@ -1,133 +1,42 @@
 #!/usr/bin/env bash
 
-# This script is intended to be run from an Ubuntu 18.04 Docker image.
-# Since it uses `apt-get` to install dependencies required by the examples,
-# it may also run on other versions of Ubuntu, or other Debian-derived distros.
-# It will definitely not run without `apt-get` installed!
+# This script is intended to be run from an Ubuntu Docker image (or other Debian-derived distros).
+# It uses `apt-get` to install dependencies required by the examples,
```

Comment:
Since the GUI examples are moving into their own repos, maybe this file should just be removed entirely?

Reply:
Yes. I think it is good to address these separately.